recipe_sha: 0f7f1f192d70eda626db447e069c77f78e4f19c3f75eff4de837be604844c857  # The commit at which this recipe failed to build.
skiplist: false # Set to true to skiplist this recipe so that it will be ignored as long as its latest commit is the one given above.
log: |-
  INFO:conda_build.utils:shutil.move(work)=/opt/conda/conda-bld/bioconductor-rcwlpipelines_1689510757243/work, dest=/opt/conda/conda-bld/bioconductor-rcwlpipelines_1689510757243/work_moved_bioconductor-rcwlpipelines-1.16.0-r43h5284365_0_linux-64_main_build_loop)
  [34mReloading output folder: [0m[34m/opt/conda/[0m[34mconda-bld[0m
  [?25l[2K[0G[] 0.0s
  [2K[1A[2K[0Gopt/conda/conda-bld/linux-64                       129.0 B @   2.6MB/s  0.0s
  opt/conda/conda-bld/noarch                         884.0 B @  32.7MB/s  0.0s
  [?25h[34mReloading output folder: [0m[34m/opt/conda/[0m[34mconda-bld[0m
  [?25l[2K[0G[] 0.0s
  opt/conda/conda-bl..    [2K[1A[2K[0Gopt/conda/conda-bld/linux-64
  opt/conda/conda-bld/noarch                         884.0 B @  36.8MB/s  0.0s
  [?25hMamba failed to solve:
   - bioconductor-rcwl >=1.16.0,<1.17.0
   - r-git2r
   - r-httr
   - r-rappdirs
   - r-base 4.3.*
   - bioconductor-biocfilecache >=2.8.0,<2.9.0
   - bioconductor-s4vectors >=0.38.0,<0.39.0
   - nodejs 14.*

  with channels:
   - conda-forge
   - bioconda
   - defaults

  The reported errors are:
  - Encountered problems while solving:
  -   - package r-base-4.3.0-h0fc540b_0 requires icu >=72.1,<73.0a0, but none of the providers can be installed
  -

  Leaving build/test directories:
    Work:
   /opt/conda/conda-bld/work
    Test:
   /opt/conda/conda-bld/test_tmp
  Leaving build/test environments:
    Test:
  source activate  /opt/conda/conda-bld/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_pl
    Build:
  source activate  /opt/conda/conda-bld/_build_env


  Traceback (most recent call last):
    File "/opt/conda/lib/python3.8/site-packages/boa/cli/mambabuild.py", line 141, in mamba_get_install_actions
      solution = solver.solve_for_action(_specs, prefix)
    File "/opt/conda/lib/python3.8/site-packages/boa/core/solver.py", line 244, in solve_for_action
      t = self.solve(specs)
    File "/opt/conda/lib/python3.8/site-packages/boa/core/solver.py", line 234, in solve
      raise RuntimeError("Solver could not find solution."  error_string)
  RuntimeError: Solver could not find solution.Mamba failed to solve:
   - bioconductor-rcwl >=1.16.0,<1.17.0
   - r-git2r
   - r-httr
   - r-rappdirs
   - r-base 4.3.*
   - bioconductor-biocfilecache >=2.8.0,<2.9.0
   - bioconductor-s4vectors >=0.38.0,<0.39.0
   - nodejs 14.*

  with channels:
   - conda-forge
   - bioconda
   - defaults

  The reported errors are:
  - Encountered problems while solving:
  -   - package r-base-4.3.0-h0fc540b_0 requires icu >=72.1,<73.0a0, but none of the providers can be installed
  -

  During handling of the above exception, another exception occurred:

  Traceback (most recent call last):
    File "/opt/conda/bin/conda-mambabuild", line 10, in <module>
      sys.exit(main())
    File "/opt/conda/lib/python3.8/site-packages/boa/cli/mambabuild.py", line 256, in main
      call_conda_build(action, config)
    File "/opt/conda/lib/python3.8/site-packages/boa/cli/mambabuild.py", line 228, in call_conda_build
      result = api.build(
    File "/opt/conda/lib/python3.8/site-packages/conda_build/api.py", line 180, in build
      return build_tree(
    File "/opt/conda/lib/python3.8/site-packages/conda_build/build.py", line 3078, in build_tree
      packages_from_this = build(metadata, stats,
    File "/opt/conda/lib/python3.8/site-packages/conda_build/build.py", line 2038, in build
      output_metas = expand_outputs([(m, need_source_download, need_reparse_in_env)])
    File "/opt/conda/lib/python3.8/site-packages/conda_build/render.py", line 787, in expand_outputs
      for (output_dict, m) in deepcopy(_m).get_output_metadata_set(permit_unsatisfiable_variants=False):
    File "/opt/conda/lib/python3.8/site-packages/conda_build/metadata.py", line 2524, in get_output_metadata_set
      conda_packages = finalize_outputs_pass(
    File "/opt/conda/lib/python3.8/site-packages/conda_build/metadata.py", line 884, in finalize_outputs_pass
      fm = finalize_metadata(
    File "/opt/conda/lib/python3.8/site-packages/conda_build/render.py", line 547, in finalize_metadata
      build_unsat, host_unsat = add_upstream_pins(m,
    File "/opt/conda/lib/python3.8/site-packages/conda_build/render.py", line 409, in add_upstream_pins
      host_deps, host_unsat, extra_run_specs_from_host = _read_upstream_pin_files(m, 'host',
    File "/opt/conda/lib/python3.8/site-packages/conda_build/render.py", line 374, in _read_upstream_pin_files
      deps, actions, unsat = get_env_dependencies(m, env, m.config.variant,
    File "/opt/conda/lib/python3.8/site-packages/conda_build/render.py", line 131, in get_env_dependencies
      actions = environ.get_install_actions(tmpdir, tuple(dependencies), env,
    File "/opt/conda/lib/python3.8/site-packages/boa/cli/mambabuild.py", line 150, in mamba_get_install_actions
      raise err
  conda_build.exceptions.DependencyNeedsBuildingError: Unsatisfiable dependencies for platform linux-64: {MatchSpec("r-base==4.3.0=h0fc540b_0"), MatchSpec("icu[version='>=72.1,<73.0a0']")}
# Last 100 lines of the build log.
